src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PAY_PER_DAY = 40;
+  const DAYS_FOR_MAX_DISCOUNT = 7;
+  const DAYS_FOR_MIN_DISCOUNT = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MIN_TERM_DISCOUNT = 20;
+
+  if (days >= DAYS_FOR_MAX_DISCOUNT) {
+    return PAY_PER_DAY * days - LONG_TERM_DISCOUNT;
```

Comment:
Create variable for `PAY_PER_DAY * days` and reuse in all places

Reply:
done ✔️